### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by zsmb13**
### 🎯 Goal

Updates the available issue types in the repository.

### 🛠 Implementation details

Will only take effect once merged to `main`, but not an urgent change.

### 🎨 UI Changes

| Before | After |
| --- | --- |
| image | image |

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [ ] ~Changelog is updated with client-facing changes~
- [ ] ~New code is covered by unit tests~
- [x] Comparison screenshots added for visual changes
- [ ] ~Affected documentation updated (docusaurus, tutorial, CMS (task created))~
- [x] Reviewers added

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/compose_task.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+TODO
+
+### Description
+
+TODO
+
+### Acceptance criteria
+
+- [] TODO
```
Comment (zsmb13) [inline .github/ISSUE_TEMPLATE/compose_task.md:19]: @filbabic Feel free to modify this one 😀

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
```
Comment (zsmb13) [inline .github/ISSUE_TEMPLATE/config.yml:1]: Enables the "Don’t see your issue here? Open a blank issue." link at the bottom of the page

Comment (petyosi) [reply .github/ISSUE_TEMPLATE/config.yml:1]: That did not work last time I tested, hopefully, it does now

**Review by petyosi — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (zsmb13) [reply .github/ISSUE_TEMPLATE/config.yml:1]: I've tried these in a private test repo and they showed up as you can see in the screenshot above.

**Review by filbabic — APPROVED** [review]

**Review by adasiewiczr — APPROVED** [review]

**Review by JcMinarro — APPROVED** [review]

**Pull request merged**